components/arm/pb_helpers.go

```diff
@@ -4,11 +4,45 @@ import (
 	"fmt"
 
 	commonpb "go.viam.com/api/common/v1"
+	pb "go.viam.com/api/component/arm/v1"
 
 	"go.viam.com/rdk/referenceframe"
 	"go.viam.com/rdk/referenceframe/urdf"
+	"go.viam.com/rdk/utils"
 )
 
+// MoveOptions define parameters to be obeyed during arm movement.
+type MoveOptions struct {
+	MaxVel, MaxAcc float64
+}
+
+func moveOptionsFromProtobuf(protobuf *pb.MoveOptions) *MoveOptions {
+	if protobuf == nil {
+		protobuf = &pb.MoveOptions{}
+	}
+
+	var vel, acc float64
+	if protobuf.MaxVelDegsPerSec != nil {
+		vel = *protobuf.MaxVelDegsPerSec
+	}
+	if protobuf.MaxAccDegsPerSec2 != nil {
+		acc = *protobuf.MaxAccDegsPerSec2
+	}
+	return &MoveOptions{
+		MaxVel: utils.DegToRad(vel),
```

Comment:
If not set, these here default to 0.

I could see a naive module implementer making <= 0 an invalid value, rather than a "use the default" value.

A comment on the struct should specify expected behavior for not-positive values.

Reply:
Yeah my intention was to have these default to zero if unset and handle default values at the driver level (per Miko's suggestion earlier in this review) I think thats a better way to handle it than trying to come up with some default at the package level.

For example if I am an arm driver I see an incoming options struct with zero values, I know then that they were unspecified since zero would result in no motion.  Or I could make the bizarre implementation choice of allowing a user to specify a zero speed.  But either way it is up to the module author here, and this code is just a passthrough.  That track with your expecations?